Makefile

```diff
@@ -76,7 +76,7 @@ DOCKER_BUILDKIT=1
 export ACK_GINKGO_DEPRECATIONS := 1.16.4
 
 # Set --output-base for conversion-gen if we are not within GOPATH
-ifneq ($(abspath $(REPO_ROOT)),$(shell go env GOPATH)/src/sigs.k8s.io/cluster-api-provider-aws)
+ifneq ($(abspath $(REPO_ROOT)),$(abspath $(shell go env GOPATH)/src/sigs.k8s.io/cluster-api-provider-aws))
```

Comment:
Why the change?

Reply:
When GOPATH has a trailing slash, the logic before my change would not detect correctly and all the generated bits would end up in a sub-directory of the repo:

```
	--output-file-base=zz_generated.conversion --output-base=/home/stevekuznetsov/code/kubernetes-sigs/cluster-api-provider-aws/src/sigs.k8s.io/cluster-api-provider-aws \
```

Reply:
Is your GOPATH manually set? If so, removing the trailing slash seems reasonable to me, we've never had anyone report any issues on this and it might break other uses, could we revert?

GOPATH from 1.8 is set by default to `$HOME/go`

Reply:
`abspath` is surjective. It does not seem possible for `$(abspath $(REPO_ROOT)` to equal some path `$foo` but NOT to equal `abspath $foo`.